test_tracetools/CMakeLists.txt

```diff
@@ -221,6 +218,37 @@ if(BUILD_TESTING)
         WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
       )
     endforeach()
+
+    # Tests to run with all instrumented/supported rmw implementations
+    set(_test_tracetools_pytest_tests_multi_rmw
+      test/test_generic_pub_sub.py
+      test/test_generic_subscription.py
+      test/test_pub_sub.py
+      test/test_publisher.py
+      test/test_subscription.py
+    )
+    set(_test_tracetools_rmw_implementations
+      rmw_connextdds
+      rmw_cyclonedds_cpp
+      rmw_fastrtps_cpp
+    )
```

Comment:
Is there a particular reason to choose these, rather than just take whatever `get_available_rmw_implementations` returns?  I don't care a huge amount right now, but as we add in more RMWs, we'll have to make additional changes here which would be nice to avoid.

Reply:
There's a comment above this section of the code that explains it:

> Tests to run with all instrumented/supported rmw implementations

Not all `rmw` implementations are instrumented. For example, `rmw_fastrtps_cpp` is , but `rmw_fastrtps_dynamic_cpp` isn't. Ideally, all core `rmw` implementations would be supported. So once that's done, we could remove this list. I think `rmw_fastrtps_dynamic_cpp` might be the only unsupported core `rmw` implementation.

We could switch this to a disallow-list. However, I believe `get_available_rmw_implementations()` returns all the `rmw` implementations it can find, so if someone has a tier 2+ implementation in their workspace and runs `test_tracetools` tests, then they will probably fail.

Let me know what you think. Maybe we don't need to be this cautious.

Reply:
All right, that's fine.  We can leave this for now, I was just wondering if we could get rid of this code.